dashboard/app/assets/javascripts/sendReport.js

```diff
@@ -96,9 +96,18 @@ function getFallbackResponse(report) {
   if (!report.fallbackResponse) {
     return null;
   }
-  return report.pass ?
-            report.fallbackResponse.success :
-            report.fallbackResponse.failure;
+  var fallbackResponse = maybeParse(report.fallbackResponse);
+  return report.pass ? fallbackResponse.success : fallbackResponse.failure;
```

Comment:
Yes, the goal is to not change the behavior except for the case where `report.fallbackResponse` is a string and can be parsed as a valid JSON object.